webcompat/helpers.py

```diff
@@ -592,9 +592,12 @@ def is_valid_issue_form(form):
     valid_form = parameters_check and values_check
     if not valid_form:
         log.info('is_valid_issue_form: form[submit_type] => {0}'.format(
-            form.get('submit_type', 'missing submit_type value')))
+            form.get('submit_type') or 'empty submit_type value'))
```

Comment:
Trying to understand which case you were trying to cover. 

```python
>>> form = {'foo': 'yeah', 'bar': 'boo'}
>>> form.get('foo', 'lol')
'yeah'
>>> form.get('foo') or 'lol'
'yeah'
>>> form = {'bar': 'boo'}
>>> form.get('foo', 'lol')
'lol'
>>> form.get('foo') or 'lol'
'lol'
>>> form = {}
>>> form.get('foo', 'lol')
'lol'
>>> form.get('foo') or 'lol'
'lol'
>>> form = {'foo': '', 'bar': 'boo'}
>>> form.get('foo', 'lol')
''
>>> form.get('foo') or 'lol'
'lol'
```

Reply:
basically this:

```
>>> f = {'a': ''}
>>> 'cool {0}'.format(f.get('a'))
'cool '
>>> 'cool {0}'.format(f.get('b'))
'cool None'
>>> 'cool {0}'.format(f.get('a') or 'dude')
'cool dude'
>>> 
```